src/aspire/utils/matrix.py

```diff
@@ -408,3 +410,47 @@ def shrink_covar(covar_in, noise_var, gamma, shrinker=None):
     shrinked_covar *= noise_var
 
     return shrinked_covar
+
+
+def do_ramping(imgs, bg_radius=1.0):
```

Comment:
Why is this in `aspire.utils.matrix`? Ideally, it should be a method of `Image`, but we have enough conflicts with #231 as it is, so let's leave it as a standalone function in `aspire.image`.

Reply:
Done in commit 90e117d.